src/TaxRate.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace PodPoint\I18n;
+
+use Mpociot\VatCalculator\VatCalculator;
+
+class TaxRate
+{
+    /**
+     * @var VatCalculator
+     */
+    private $vatCalculator;
+
+    public function __construct(VatCalculator $vatCalculator)
+    {
+        $this->vatCalculator = $vatCalculator;
+    }
+
+    /**
+     * Returns the tax rate for the given country code.
+     *
+     * @param string $countryCode
+     *
+     * @return float
+     */
+    public function get(string $countryCode): float
```

Comment:
Out of curiosity why you are adding return type hinting here but not in other places?

Reply:
You're right. Inconsistency. Removing them for now until we settle on some standards as a team enforced with linting and CI pipelines.

Reply:
Please keep then!
I was just asking why?
They are far more important than doc blocks and they will actually cast return type.

Reply:
Added them everywhere. I still think docblocks can let us do more like `array|null`. We'll need to wait until we upgrade all our stacks to php 7.3+ to get a better support of typehinting IMO.

Reply:
you can do array|null even now?
`?array $someArray = null`
I think atm you cannot do things you should not be doing?